modules/Authentication.php

```diff
@@ -49,18 +77,12 @@ public static function withEmailPassword($db, $email, $password)
         return null;
     }
 
-    public function getUserIdFromSessionToken($session_token)
-    {
-        $stmt = $this->db->prepare("SELECT * FROM `user_session` WHERE `session_token` = ?");
-        $stmt->bindParams('i', $session_token);
-        $stmt->execute();
-        $result = $stmt->getResults();
-        $row = $result->fetchAssoc();
-        $stmt->close();
-        return $row;
-    }
-
-    // Create session token and adding log in database for validation
+    /**
+     * @method : createSession
+     * @description : Create session token and adding log in database for validation
+     * @db MysqlDatabase: database instance
+     * @user_id int : user_id of the user
+     */
     public static function createSession($db, $user_id)
     {
         if ($user_id > 0) {
```

Comment:
```
$stmt->execute();
$affected_rows = $stmt->getAffectedRows();
$stmt->close();
```
These lines look redundant in the `if else` statement. You could extract these three lines outside the conditional to prevent code repetition.